bridge/client/app/_components/ktb-modify-uniform-subscription/ktb-modify-uniform-subscription.component.spec.ts

```diff
@@ -29,7 +33,7 @@ describe('KtbModifyUniformSubscriptionComponent', () => {
       })
     );
     await TestBed.configureTestingModule({
-      imports: [AppModule, HttpClientTestingModule],
+      imports: [KtbModifyUniformSubscriptionModule, HttpClientTestingModule, RouterTestingModule, AppRouting],
```

Comment:
There's actually a better option than to load all views with `AppRounting`:

https://github.com/keptn/keptn/pull/8127/files#diff-9a748d3b7e6e369ca85b32ec19a3a63bdf6e75dfb5e97dea2b1c9cba7a35c1e2R29

The issue with including `AppRounting` is that the routes will eventually disappear in there as soon as we start the lazy loading migration. Better include only the necessary routes, the actual component they are routing to is totally irrelevant (the `RouterTestingModule` will not actually route there in the test).